packages/grafana-ui/src/components/PanelChrome/PanelChrome.tsx

```diff
@@ -124,6 +139,18 @@ export function PanelChrome({
           </h6>
         )}
 
+        {panelNotices && <PanelNotices notices={panelNotices.getPanelNotices()} onClick={panelNotices.onClick} />}
+
+        <PanelDescription description={description} />
+
+        <PanelLinks links={links} />
+
+        {titleItems && (
+          <div className={styles.titleItems} data-testid="title-items">
```

Comment:
I think panel notices and links could just use regular title items, don't think we need to treat them differently and would make this component a bit simpler